Core/src/org/sleuthkit/autopsy/modules/stix/STIXReportModule.java

```diff
@@ -101,6 +103,13 @@ public void generateReport(String baseReportDir, ReportProgressPanel progressPan
         progressPanel.setIndeterminate(false);
         progressPanel.start();
         progressPanel.updateStatusLabel(NbBundle.getMessage(this.getClass(), "STIXReportModule.progress.readSTIX"));
+        String stixReportDir = baseReportDir + " " + STIX_REPORT;
```

Comment:
The individual report modules should not modify the base report name passed in to the generateRpeort method because the reports do not "own" this path, so modifications to it could break something in the future. What we should do instead is have the ReportGenerator call ReportModule.getName and insert it into the base report path between the the case name and the time stamp. This means that ReportGenerator will no longer be able to construct the complete report path in its constructor. I would create a format string in the constructor instead, and then use it to make the actual base report path in the four places that it is used. In each of these places, there is a ReportModule that can be used for the call to ReportModule.getName. In connection with this, we should make sure all of our implementations of ReportModule,getName return a reasonable value for use in the base path as an aid to users finding the report they want. It looks like several of the getName implementations return a string that starts with "Results - ", which is not particularly helpful these days, so it can be discarded.